src/shared/TeamPage/TeamPage.tsx

```diff
@@ -0,0 +1,72 @@
+import Modal from '@material-ui/core/Modal';
+import { makeStyles, Theme, createStyles } from '@material-ui/core/styles';
+import { DataGrid } from '@material-ui/data-grid';
+import AssignmentIcon from '@material-ui/icons/Assignment';
+import React, { useEffect, useState} from 'react';
+
+
+import { Person } from 'models/registry.models';
+
+import classes from './TeamPage.module.css';
+
+interface Rows extends Person {
+  id?: string | number;
+  lp?: string | number;
+}
+
+interface Props {
+  members: Rows[];
+}
+
+const TeamPage = ({ members } : Props): JSX.Element => {
+  const [isOpen, setOpen] = React.useState<boolean>(false);
+  const [rows, setRows] = useState<any>([]);
+  const columns = [
+    { field: 'lp', headerName: 'LP', width: 80, },
+    { field: 'name', headerName: 'First name', width: 150 },
+    { field: 'surname', headerName: 'Surname', width: 150 }
+  ];
+
+  const handleOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  useEffect(() => {
+    const helper = members ? (members.map((el, index) => {
```

Comment:
"helper" is not correct name for this variable. We use setRows, so we can expect these to be "rows"